2.2/.sti/bin/assemble

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function rake_assets_precompile() {
+  [ -n "$DISABLE_ASSET_COMPILATION" ] && return
+  [ ! -f Gemfile ] && return
+  [ ! -f Rakefile ] && return
+  ! grep " rails " Gemfile.lock >/dev/null && return
+  ! grep " execjs " Gemfile.lock >/dev/null && return
+  ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
+
+  echo "---> Starting asset compilation."
+  bundle exec rake assets:precompile
+}
+
+set -e
+
+export RACK_ENV=${RACK_ENV:-"production"}
+export RAILS_ENV=${RAILS_ENV:-"${RACK_ENV}"}
```

Comment:
Same as my previous comment: I've we are going to split the ruby and ror images we dont need this to be set here, but in the ror image